### ISSUE TITLE
Add `Kb` and `KB`

### ISSUE THREAD

**Issue by carlocorradini** [jonahsnider/convert#547]
Add `Kb` and `KB` to data digital units.

**Comment by jonahsnider** [comment]
"Kb" and "KB" are not units of data. The SI symbol for "kilo-" is "k", not "K".

**Comment by carlocorradini** [comment]
Agree, however, for digital units, the capital `K` is the standard.

**Comment by jonahsnider** [comment]
This is not 100% true. There are two main standards for defining digital data units, SI and IEC. I forgot about the IEC standard 
when I wrote my previous response, but after reading a bit about it I changed my mind & will add the `Kb` and `KB` units to Convert.

SI states that a `kB` is `8000` bytes and IEC says that a `KB` is `8192` bytes. Convert currently supports the IEC units `KiB`, `MiB`, `GiB`, `TiB`, and `PiB`, but does not include `KB`, `MB`, `GB`, `TB`, or `PB`.

I think it makes sense to add the `Kb` and `KB` units as aliases for `kb` and `kB`, rather than the actual IEC definitions. Having `kb` be a different value than `Kb` seems like bad library design and setting users up for failure later on.
